foghorn-edge/templates/analytics-engine-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
License and Copyright information is missing.
open-ness/edgeapps repo hosts the open source edge applications. The license must be open source License.